src/logger/UsageLoggers.go

```diff
@@ -0,0 +1,63 @@
+package logger
+
+import (
+	"sync"
+	//Library used for getting environment variables and other useful env things
+	"os"
+	//used for converting the string returned by lookupEnv
+	"strconv"
+)
+
+//name change since helper uses the name "once"
+var onceUsageLoggers sync.Once
+
+type UsageLoggers struct {
+	bricked  bool
+	disabled bool
+}
+
+var usageLoggers *UsageLoggers
+
+func GetUsageLoggers() *UsageLoggers {
+	onceUsageLoggers.Do(func() {
+		envLookup, _ := os.LookupEnv("USAGE_LOGGERS_DISABLE")
+		_bricked, err := strconv.ParseBool(envLookup)
+		if err != nil {
+			//help????
```

Comment:
I'm thinking we should have `GetUsageLoggers()` return and error as well so `(*UsageLoggers, error)` we can have an informative error letting the Devops member know that the environment variables weren't set up correctly.

Reply:
Thoughts?